wandb/data_types.py

```diff
@@ -220,6 +233,10 @@ def bind_to_run(self, run, key, step, id_=None):
             extension = self._extension
             rootname = os.path.basename(self._path)[: -len(extension)]
 
+        with open(self._path, "rb") as f:
+            self._sha256 = hashlib.sha256(f.read()).hexdigest()
```

Comment:
Ah, yup, these sha256 operations are expensive. Especially since we don't parallelize them. Not your fault and let's leave in for now.

Reply:
good to know. ty